### PR TITLE
pip install papermill

### PR DESCRIPTION
**Pull request by djherbis**
papermill provides a bunch of nice features for executing notebooks, including ones not provided by nbconvert.

Some key features we want:
* dump cell output to logs
* stop on cell error, but still include cell output up to the errored cell

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -487,6 +487,9 @@ RUN jupyter-nbextension disable nb_conda --py --sys-prefix && \
     jupyter-nbextension disable nbpresent --py --sys-prefix && \
     jupyter-serverextension disable nbpresent --py --sys-prefix
 
+# Install papermill so it can replace nbconvert as a method to execute notebooks.
+RUN pip install papermill
```
Comment (rosbo) [inline Dockerfile:491]: Is it worth creating its own Docker layer for this package? Consider adding it at line 424.

If you do have to run this command at the end, make sure to call the `/tmp/clean-layer.sh` script to reduce the layer size.

Comment (djherbis) [reply Dockerfile]: Moved thanks :)

**Comment by djherbis** [comment]
Working on a test.

**Comment by djherbis** [comment]
> Working on a test.

Done.

**Comment by rosbo** [comment]
Please wait until the build & tests succeed before merging. Thanks

**Pull request merged**